sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -773,7 +774,7 @@ class DDLParserSuite extends AnalysisTest with SharedSparkSession {
   }
 
   test("create table - basic") {
-    val query = "CREATE TABLE my_table (id int, name string)"
+    val query = "CREATE TABLE my_table (id int, name string) STORED AS textfile"
```

Comment:
shall we update the test assertions? We do want to test the basic CREATE TABLE command and make sure the created table is a native parquet table.

Reply:
Good idea!